rbac/management/role/definer.py

```diff
@@ -67,7 +67,7 @@ def _update_or_create_roles(tenant, roles):
     """Update or create roles from list."""
     for role_json in roles:
         try:
-            _make_role(tenant, role_json)
```

Comment:
Where will this return? Will it break the for loop?

Reply:
Nice catch, Jay! I added a new commit that ensures we collect all IDs.